app/src/main/java/com/architectcoders/aacboard/network/response/ArasaacPictogramDTO.kt

```diff
@@ -1,18 +1,25 @@
 package com.architectcoders.aacboard.network.response
 
-import com.architectcoders.aacboard.domain.data.arasaac.ArasaacKeyword
-import com.architectcoders.aacboard.domain.data.arasaac.ArasaacPictogram
+import com.architectcoders.aacboard.domain.data.cell.CellPictogram
+import com.google.gson.annotations.SerializedName
 
-data class ArasaacPictogramDTO(
-    val id: Int,
-    val arasaacKeywords: List<ArasaacKeyword>,
-    val sex: Boolean,
-    val violence: Boolean,
-) {
-    fun toArasaacPictogram() = ArasaacPictogram(
-        id = this.id,
-        arasaacKeywords = this.arasaacKeywords,
-        sex = this.sex,
-        violence = this.violence,
-    )
-}
\ No newline at end of file
+    data class  ArasaacPictogramDTO(
+        @SerializedName("_id")
+        val id: Int,
+        val keywords: List<ArasaacKeywordDTO>,
+        val sex: Boolean,
```

Comment:
I think we could remove the `sex` and `violence` fields, we are not using them

Reply:
ok